proto/types/action.proto

```diff
@@ -225,25 +236,8 @@ message ActionCore {
   uint32 chainID = 5;
   oneof action {
     Transfer transfer = 10;
+    BlobTx blobTx = 11;
     Execution execution = 12;
-    // FedChain
-    StartSubChain startSubChain = 13;
-    StopSubChain stopSubChain = 14;
-    PutBlock putBlock = 15;
-    CreateDeposit createDeposit = 16;
-    SettleDeposit settleDeposit = 17;
-
-    // PlumChain
-    CreatePlumChain createPlumChain = 18;
-    TerminatePlumChain terminatePlumChain = 19;
-    PlumPutBlock plumPutBlock = 20;
-    PlumCreateDeposit plumCreateDeposit = 21;
-    PlumStartExit plumStartExit = 22;
-    PlumChallengeExit plumChallengeExit = 23;
-    PlumResponseChallengeExit plumResponseChallengeExit = 24;
-    PlumFinalizeExit plumFinalizeExit = 25;
-    PlumSettleDeposit plumSettleDeposit = 26;
-    PlumTransfer plumTransfer = 27;
```

Comment:
these are added at very beginning, but never implemented or used.
Would rather remove them for clarity